### ISSUE TITLE
What else is needed to publish this sim?

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/coulombs-law#62]
@mbarlow12 @ariel-phet @arouinfar since we are going to proceed with publishing this sim without any a11y, is there anything else that needs to be done before we proceed to an RC?

**Comment by jessegreenberg** [comment]
@mbarlow12 was coulombs-law code reviewed? I recall that inverse-square-law-common was, which holds most of the sim's implementation.

**Comment by arouinfar** [comment]
@jessegreenberg this sim was code reviewed in fall 2017 (#33), but that predates the ISLC code review (https://github.com/phetsims/inverse-square-law-common/issues/30) that followed in spring 2018. I'd like to review the dev version that was published yesterday to give it my final sign-off before it moves onto QA testing.

**Comment by arouinfar** [comment]
@jessegreenberg dev.18 looks good to me! I think this sim is ready for QA (assuming the CL code review predating the ISLC code review is a non-issue).

**Comment by jessegreenberg** [comment]
I think https://github.com/phetsims/coulombs-law/issues/33 was for personal  review prior to a dev test. I don't think this repo has been reviewed by anyone other than @mbarlow12. I reviewed inverse-square-law-common and gravity-force-lab-basics and would be happy to review this repo too.

@ariel-phet can you please comment on priority and if this is something I or another developer should do?

**Comment by ariel-phet** [comment]
@jessegreenberg yes, this is something another dev should do, I will mark for dev meeting (your time is a bit overbooked at the moment). 

Perhaps @Denz1994 could do a code review with @chrisklus riding along?

**Comment by ariel-phet** [comment]
Marking for dev meeting so we can see who is available (and excited!) to do a review

**Comment by mbarlow12** [comment]
Sorry for the late input. The only items on the radar are a code review and to keep an eye on the ruler dragging issue (https://github.com/phetsims/coulombs-law/issues/41). We decided to move forward without specifically addressing it, but I could easily see this coming up in either the review or during RC/QA testing.

**Comment by jessegreenberg** [comment]
Sounds good. It was decided that this 1.0 for this sim would be deployed without a11y, so #41 could be deferred till the next release.

**Comment by samreid** [comment]
Should we implement PhET-iO for testing/publication at the same time? Some progress made in instrumenting inverse-square-law-common was made in https://github.com/phetsims/gravity-force-lab/issues/76

**Comment by samreid** [comment]
@ariel-phet requested @Denz1994 to code review, once M&S is published.  @mbarlow12  please create a code review issue and assign @Denz1994 and @chrisklus with the intention that @Denz1994 will lead the code review and @chrisklus may tag along.

**Comment by mbarlow12** [comment]
Code review created: https://github.com/phetsims/coulombs-law/issues/63.

**Comment by mbarlow12** [comment]
#63 is now complete, and we're ready to move on to dev testing. See https://github.com/phetsims/QA/issues/203 for details.

New dev release at:
https://phet-dev.colorado.edu/html/coulombs-law/1.0.0-dev.19/phet/coulombs-law_en_phet.html

**Comment by arouinfar** [comment]
Given that we're now in rc testing (yay!!), I think we can go ahead and close this issue.
